test/model/src/driver/kv_workload_generator.cpp

```diff
@@ -76,6 +86,17 @@ kv_workload_generator_spec::kv_workload_generator_spec()
     nonprepared_transaction_rollback = 0.1;
     prepared_transaction_rollback_after_prepare = 0.1;
     prepared_transaction_rollback_before_prepare = 0.1;
+
+    timing_stress_ckpt_slow = 0.2;
```

Comment:
The probabilities add to more than 1, so only the first five settings would be actually used.

Reply:
Fixed in b15a4f491906d1b92d4955cb07a321633d9f7722